.github/workflows/pkg.pr.new.yml

```
@@ -6,33 +6,44 @@ on:
     types: [submitted]
 
 jobs:
-  deploy-preview:
-    name: Deploy to pkg.pr.new
-    if: github.event.review.state == 'APPROVED' || contains(github.event.pull_request.author_association, 'MEMBER')
+  check-changes:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332
         with:
           fetch-depth: 2
-      - name: Check for changes to nuqs package
-        id: check-for-changes
+
+      - name: Check for PR Approval
         run: |
-          if git diff --quiet HEAD^ HEAD ./packages/nuqs; then
-            echo "No changes to nuqs package, skipping preview deployment."
-            echo "skip=true" >> $GITHUB_OUTPUT
+          if [[ "${{ github.event.review.state }}" != "APPROVED" && ! "$(echo "${{ github.event.pull_request.author_association }}" | grep 'MEMBER')" ]]; then
```

Restore the Git check

```suggestion
          if git diff --quiet HEAD^ HEAD ./packages/nuqs; then
```